### DISCUSSION TITLE
A lot of the sensor alternative missing from UI setup

### DISCUSSION THREAD

**Discussion by filikun** [dolezsa/thermal_comfort#215] [General]
The documentation lists a lot of sensors that are missing from the list below. Do I need something more for them to show?

![image](https://user-images.githubusercontent.com/9947929/196762623-529ed44e-cd4f-4adb-aaea-1ccaa84f58f8.png)

**Comment by rautesamtr** [comment]
They are currently in master and will be released with 2.0
